server/src/main/java/org/apache/iotdb/db/engine/compaction/cross/utils/PointElement.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.db.engine.compaction.cross.utils;
+
+import org.apache.iotdb.tsfile.read.common.IBatchDataIterator;
+import org.apache.iotdb.tsfile.read.reader.chunk.ChunkReader;
+import org.apache.iotdb.tsfile.utils.Pair;
+
+import java.io.IOException;
+
+public class PointElement {
+  public long timestamp;
+  public int priority;
+  public Pair<Long, Object> timeValuePair;
```

Comment:
Use TimeValuePair instead of Pair

Reply:
The type of value in TimeValuePair is `TsPrimitiveType`, and the type returned by BatchDataIterator is `Object`. If  change this attribute to `TimeValuePair`, you need to convert `Object` type to `TsPrimitiveType` type every time you read a point, which is costly.